code/tests/test_app.py

```diff
@@ -245,10 +244,11 @@ def test_conversation_custom_returns_correct_response(
         get_message_orchestrator_mock.return_value = message_orchestrator_mock
 
         env_helper_mock.AZURE_OPENAI_MODEL = self.openai_model
+        env_helper_mock.CONVERSATION_FLOW = "custom"
```

Comment:
Is there a reason why you didn't set this as the default above?

Reply:
good idea. made the change.